src/cmds/pbs_attach.c

```diff
@@ -231,7 +196,9 @@ win_attach(BOOL use_cmd, int newsid, int port, int doparent, pid_t pid, char *jo
 }
 #endif
 int
-main(int argc, char *argv[])
+main(argc, argv)
```

Comment:
The older function was ansi compliant. Are you working on an older copy of this file?